data/defikingdoms_itemtogold/dfk_item_to_gold.csv

```diff
@@ -0,0 +1,18 @@
+﻿CONTRACT_ADDRESS,NPC_VERSION,GOLD
+0x043f9bd9bb17dfc90de3d416422695dd8fa44486,1,3.0
+0x094243dfabfbb3e6f71814618ace53f07362a84c,1,15.0
+0x2493cfdacc0f9c07240b5b1c4be08c62b8eeff69,1,100.0
+0x372caf681353758f985597a35266f7b330a2a44d,1,60.0
+0x600541ad6ce0a8b5dae68f086d46361534d20e80,1,100.0
+0x68ea4640c5ce6cc0c9a1f17b7b882cb1cbeaccd7,1,10.0
+0x6b10ad6e3b99090de20bf9f95f960addc35ef3e2,1,5.0
+0x6e1bc01cc52d165b357c42042cf608159a2b81c1,1,12.5
+0x78aed65a2cc40c7d8b0df1554da60b38ad351432,1,3.0
+0x8bf4a0888451c6b5412bcad3d9da3dcf5c6ca7be,1,5.0
+0xb80a07e13240c31ec6dc0b5d72af79d461da3a70,1,50.0
+0xcdffe898e687e941b124dfb7d24983266492ef1d,1,75.0
+0xe4cfee5bf05cef3418da74cfb89727d8e4fee9fa,1,5.0
+0xac5c49ff7e813de1947dc74bbb1720c353079ac9,1,5.0
+0xc0214b37fcd01511e6283af5423cf24c96bb9808,1,12.5
+0x19b9f05cde7a61ab7aae5b0ed91aa62ff51cf881,1,10.0
+0xc5891912718ccffcc9732d1942ccd98d5934c2e1,1,15.0
```

Comment:
add a line 
where's the source of this csv?

Reply:
Gooood question.. lol… let me back track.